osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/SpectatorSyncManager.cs

```diff
@@ -30,41 +30,57 @@ public class CatchUpSyncManager : Component, ISyncManager
         /// </summary>
         public const double MAXIMUM_START_DELAY = 15000;
 
+        /// <summary>
+        /// An event which is invoked when gameplay is ready to start.
+        /// </summary>
         public event Action? ReadyToStart;
 
         /// <summary>
-        /// The master clock which is used to control the timing of all player clocks clocks.
+        /// The catch-up state of the master clock.
         /// </summary>
-        public GameplayClockContainer MasterClock { get; }
-
         public IBindable<MasterClockState> MasterState => masterState;
 
+        public double CurrentMasterTime => masterClock.CurrentTime;
+
+        /// <summary>
+        /// The master clock which is used to control the timing of all player clocks clocks.
+        /// </summary>
+        private GameplayClockContainer masterClock { get; }
```

Comment:
`readonly`?

Reply:
It's get-only so roughly equivalent, but can change if it's preferred. Dunno if we really have a standard for this.

Reply:
I personally use readonly fields for private members, and get-only properties for protected/public accessibility.

Reply:
I didn't mean this to start a discussion. I'm OCD and considered this an error because the original method was `public { get; }` _only_ because it was defined in the interface.

Reply:
yeah i also prefer `readonly`, thus the change being made